### DISCUSSION TITLE
Appearently its not possible to get headers if they are defined multiline

### DISCUSSION THREAD

**Discussion by ta3pks** [sagebind/isahc#122] [Questions] [answered]
I am trying to communicate with an API which sets multiple `Set-Cookie` headers but i am able to only read the first one using 
```
        res.headers()
                .get("Set-Cookie")
                .map(|val| val.to_str().unwrap_or_default())
                .unwrap_or_default()
                .to_string(),
```

**Comment by sagebind** [comment, marked as answer]
Try using [`get_all()`](https://docs.rs/http/0.1.19/http/header/struct.HeaderMap.html#method.get_all); you should see each instance of the response header as a separate value.
